day_7/ceasar_cipher.rb

```diff
@@ -0,0 +1,22 @@
+def shift_char(c, base, offset)
+  (((c.ord - base) + offset) % 26 + base).chr
```

Comment:
Nice use of `.ord` in this helper method here. This is definitely the crux of shifting and you nailed it.